apps/system/test/unit/telephony_monitor_test.js

```diff
@@ -0,0 +1,41 @@
+/* global BaseModule, MockNavigatorMozTelephony */
+'use strict';
+
+require('/shared/test/unit/mocks/mock_navigator_moz_telephony.js');
+requireApp('system/js/service.js');
+requireApp('system/js/base_module.js');
+requireApp('system/js/telephony_monitor.js');
+
+suite('system/TelephonyMonitor', function() {
+  var subject;
+
+  setup(function() {
+    subject = BaseModule.instantiate('TelephonyMonitor', 
```

Comment:
nit: I think there's a trailing space here :)
(our linter doesn't raise this anymore I think :/)